components/cqrs/event_processor_group.go

```diff
@@ -0,0 +1,260 @@
+package cqrs
+
+import (
+	stdErrors "errors"
+	"fmt"
+
+	"github.com/ThreeDotsLabs/watermill"
+	"github.com/ThreeDotsLabs/watermill/message"
+	"github.com/pkg/errors"
+)
+
+type EventGroupProcessorConfig struct {
+	// GenerateSubscribeTopic is used to generate topic for subscribing to events for handler groups.
+	// This option is required for EventProcessor if handler groups are used.
+	GenerateSubscribeTopic EventGroupProcessorGenerateSubscribeTopicFn
+
+	// SubscriberConstructor is used to create subscriber for GroupEventHandler.
+	// This function is called for every events group once - thanks to that it's possible to have one subscription per group.
+	// It's useful, when we are processing events from one stream and we want to do it in order.
+	SubscriberConstructor EventGroupProcessorSubscriberConstructorFn
+
+	// OnHandle works like OnHandle, but is called for group handlers instead.
+	// OnHandle is not called for handlers group.
```

Comment:
Is this doc correct? I'm not sure I get what it means.

Reply:
leftover from old structure, removed